frontend/workflows/serverexperimentation/src/start-experiment.tsx

```diff
@@ -12,8 +12,58 @@ import {
 import { useDataLayout } from "@clutch-sh/data-layout";
 import type { WizardChild } from "@clutch-sh/wizard";
 import { Wizard, WizardStep } from "@clutch-sh/wizard";
+import { FormControl, FormControlLabel, FormLabel, Radio, RadioGroup } from "@material-ui/core";
 import * as yup from "yup";
 
+interface RadioControlItem {
+  label: string;
+  value: string;
+}
+
+interface RadioControlProps {
+  name: string;
+  label: string;
+  items: RadioControlItem[];
+  onChange: (value: string) => void;
+}
+
+const RadioControl: React.FC<RadioControlProps> = ({ name, label, items, onChange }) => {
```

Comment:
@dschaller do you think I could move this control to Core instead?

Reply:
Of course :D